singer_sdk/sinks/sql.py

```diff
@@ -282,7 +282,7 @@ def generate_insert_statement(
         statement = dedent(
             f"""\
             INSERT INTO {full_table_name}
-            ({", ".join(property_names)})
+            ({", ".join([f'"{name}"' for name in property_names])})
```

Comment:
We could try using https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.quoted_name

Reply:
I'm not typically a python developer, but that change looks to be easy enough. I'll make sure to get those tests fixed up too.